rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,150 @@
+### DCP PR:
+***Leave this blank until the RFC is approved then the Author(s) must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc\#\](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)`
+
+# User Support SOP
+
+## Summary
+
+This process RFC describes how the DCP handles user support requests through the use of the Zendesk application and other processes.
+
+## Author(s)
+
+`[Kevin Osborn](mailto:kosborn2@ucsc.edu)`
+
+`[Jodi Hirschman](mailto:jhirsch@broadinstitute.org)`
+
+`[Matthew Spier](mailto:mspeir@ucsc.edu)`
+
+`[Mallory Freeberg](mailto:mfreeberg@ebi.ac.uk)`
+
+## Shepherd
+
+***Leave this blank. This role is assigned by DCP PM to guide the Author(s) through the RFC process.*
+
+## Motivation
+
+### Immediate Need 
+Users must have a stadard way to guve feedback and request help. There will be a general up-tick on support demand every time we release a new feature or roll out a new type of data. Having a basic ticketing system in place will help assure that no problems are missed. Understanding the needs of the DCP users is paramount. 
+
+### Sustainability Through the GA
+Upon general release, the full range of use cases will come into play and we anticipate a large increase in both the nubmer of tickets and diversity of questions asked by users. Addressing these tickets will require a big time committment, so we need an efficient system in place and a team well-versed in the DCP and its scientific applications. 
+
+This RFC describes an SOP that we think will satisfy both the Beta2 test phase and the early days of the GA. As we grow in our understanding of user support needs, this SOP will likely need revision to address long-term sustainability.
```

Comment:
I think that the RFC should avoid referencing _GA_.

Reply:
GA has taken on many meanings to many people, but ultimately it means when the prod.data.humancellatlas.org becomes data.humancellatlas.org. We will be redirecting the Preview site to that address at the same time. This means that the DCP will be generally available. I will add some text to clarify what I mean by GA to the PR.  Let me know if you still object. (making several changes at once so give me some time.)